cmd/wio/toolchain/npm/resolve/types.go

```diff
@@ -22,11 +23,14 @@ type Info struct {
 }
 
 type Node struct {
-    name string
-    ver  string
-    deps []*Node
-
-    resolve *semver.Version
+    Name            string
+    ConfigVersion   string
+    ResolvedVersion *semver.Version
+    Dependencies    []*Node
+
+    Vendor bool
```

Comment:
Why is this field needed again? Is the path itself to the project no sufficient?